includes/admin.inc

```diff
@@ -108,21 +105,21 @@ function fb_instant_articles_module_config_ads($form) {
     '#element_validate' => array('fb_instant_articles_validate_an_placement_id'),
     '#states' => array(
       'visible' => array(
-        '[name=fb_instant_articles_ad_type]' => array('value' => FB_INSTANT_ARTICLES_AD_TYPE_FBAN),
+        '[name=fb_instant_articles_ad_type]' => array('value' => AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_FBAN),
       ),
     ),
   );
 
-  $form['ads']['fb_instant_articles_ads_dimension'] = array(
+  $form['ads']['fb_instant_articles_ads_dimensions'] = array(
     '#type' => 'select',
     '#title' => t('Ad Dimensions'),
     '#options' => array(
       0 => t('Large (300 x 250)'),
     ),
-    '#default_value' => variable_get('fb_instant_articles_ads_dimension'),
+    '#default_value' => variable_get('fb_instant_articles_ads_dimensions'),
```

Comment:
Also, should be consistent with the default value here, going by the changes further down, should be:

```
variable_get('fb_instant_articles_ads_dimensions', AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE)
```